examples/flask/hello.py

```diff
@@ -99,126 +181,57 @@ def get_tasks():
     }
 ]
 
+
 @app.route('/test/html_response', methods=['GET'])
 def html_response():
-    html_text = """
+    html_body = """
         <!DOCTYPE html>
         <html>
         <body>
-    
+
         <h1>My First Heading</h1>
-    
+
         <p>My first paragraph.</p>
-    
+
         </body>
         </html>
         """
-    return Response(response=html_text, status=200, mimetype='text/html')
+    return Response(response=html_body, status=201, mimetype='text/html')
```

Comment:
original status was 200 - new one is 201?

Reply:
changed